.gitlab-ci.yml

```diff
@@ -17,7 +17,7 @@ test:
         - apt-get install -y cmake xorg-dev mesa-utils # glfw
     script:
       - mkdir -p $JULIA_DEPOT_PATH # Pkg.jl#325
-      - julia --project -e 'using Pkg; Pkg.build(); Pkg.test()'
+      - julia --project -e 'using Pkg; Pkg.build(); Pkg.test(; coverage=true)'
```

Comment:
Well, we don't upload coverage, since it doesn't make any sense for Makie... And since there is a cost to recording coverage, I purposely disabled it^^

Reply:
I think you enabled it in 61fb64908302bd318c41514d909073c0a7e84e04?